public/app/plugins/datasource/cloud-monitoring/datasource.ts

```diff
@@ -134,10 +106,19 @@ export default class CloudMonitoringDatasource extends DataSourceApi<CloudMonito
       .toPromise();
   }
 
+  applyTemplateVariables(target: CloudMonitoringQuery, scopedVars: ScopedVars): Record<string, any> {
+    target = this.prepareTimeSeriesQuery(target, scopedVars);
+    return {
+      ...target,
+      intervalMs: this.intervalMs,
+      type: 'timeSeriesQuery',
+    };
+  }
+
   getTimeSeries(options: DataQueryRequest<CloudMonitoringQuery>): Observable<PostResponse> {
```

Comment:
Looks like function is not needed anymore since it's only used by getLabels, so move the logic to getLabels.